packages/create-itp-react-app/templates/app/config-overrides.js

```diff
@@ -3,17 +3,17 @@ const rewirePostCss = require('react-app-rewire-postcss');
 
 module.exports = {
   webpack: (config, env) => {
-    config = rewireCssModules.webpack(config, env, {
+    let newConfig = rewireCssModules.webpack(config, env, {
       test: /\.module.css$/
     });
 
-    config = rewirePostCss(config, {
-      plugins: loader => [
-        require('postcss-preset-env')(),
+    newConfig = rewirePostCss(newConfig, {
+      plugins: loader => [ // eslint-disable-line no-unused-vars
```

Comment:
why not just `() => []` instead?

Reply:
Eh, I kinda like the verbosity here, makes it clear what the code is about. I can remove it though. :)

Reply:
fine by me